### ISSUE TITLE
[BITV]: aria-label for "select all" for files table header not on interactive element

### ISSUE THREAD

**Issue by susnux** [nextcloud/server#41713]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

The files table header "select all" should have a label, currently the label is provided through "aria-label" attribute but is is set on the `span` element rather then the `input` or `label`.
`aria-label` must be set on interactive elements, so this is an accessibility problem.

**Comment by JuliaKirschenheuter** [comment]
Already done?

![Image](https://github.com/nextcloud/server/assets/6078378/cd406e60-6f9b-4332-9c4d-97d3d310e16a)

**Comment by szaimen** [comment]
Apparently resolved in vue 8.4.0: https://github.com/nextcloud-libraries/nextcloud-vue/commit/85bb4722d541fd2824948637ac72cfff06e5fc32

**Comment by JuliaKirschenheuter** [comment]
https://github.com/nextcloud/server/issues/41713#issuecomment-1888688961

**Comment by szaimen** [comment]
changed to close as not planned
